qutebrowser/browser/commands.py

```diff
@@ -1116,12 +1116,32 @@ def quickmark_load(self, name, tab=False, bg=False, window=False):
         self._open(url, tab, bg, window)
 
     @cmdutils.register(instance='command-dispatcher', scope='window')
-    def bookmark_add(self):
-        """Save the current page as a bookmark."""
+    def bookmark_add(self, url=None, title=None):
+        """Save the current page as a bookmark, or a specific url.
+
+        If no url and title are provided, then save the current page as a
+        bookmark.
+        If a url and title have been provided, then save the given url as
+        a bookmark with the provided title.
+
+        Args:
+            url: url to save as a bookmark. If None, use url of current page.
+            title: title of the new bookmark."""
```

Comment:
Yeah, ideally I would have run pylint on my machine but the tox command seemed to take ages (it's an old laptop I'm using).